sidebar.less

```diff
@@ -183,22 +183,22 @@
 	// Subscribe buttons (RES wraps in .subButtons)
 	.subscribe-button {
 		order: 1;
-		
+		margin-right: 0px;
 		text-align: center;
 		
 		a {
 			width: 130px;
 		}
 	}
 	
-	.subscribe-button a.active,					// Normal subscribe button
+	.subscribe-button a.active,			// Normal subscribe button
```

Comment:
You must be using a different indentation size (I use 4), in which case you actually messed up the alignment:

![](http://i.imgur.com/WYTQ8Sb.png)

I thought I had set up an editorconfig, but I guess I didn't.